cognoml/analysis.py

```diff
@@ -85,6 +86,11 @@ def classify(sample_id, mutation_status, **kwargs):
         performance[part] = utils.value_map(metrics, round, ndigits=5)
     performance['cv'] = {'auroc': round(clf_grid.best_score_, 5)}
     results['performance'] = performance
+    
+    results['model'] = utils.model_info(clf_grid.best_estimator_)
+
+    feature_df = utils.get_feature_df(clf_grid.best_estimator_, X.columns)
```

Comment:
@yl565 currently I'm retrieving feature names using `X.columns` which will get the feature names of `X` before it enters the pipeline. However, since `VarianceThreshold` or other feature selection/tranformation steps will alter the feature set, do you know how we can get feature names at the end of the pipeline? In other words, we want the feature names corresponding to `clf_grid.best_estimator_.coef_`. I searched for like an hour and couldn't figure this out.